.travis.yml

```diff
@@ -18,16 +18,19 @@ cache:
 before_script:
   - (which cargo-install-update && cargo install-update cargo-update) || cargo install cargo-update
   - (which cargo-prune && cargo install-update cargo-prune) || cargo install cargo-prune
-  - if [ "${TRAVIS_RUST_VERSION}" = stable ]; then
-      (which rustfmt && cargo install-update rustfmt) || cargo install rustfmt;
+  - if [ "${TRAVIS_RUST_VERSION}" = 1.17.0 ]; then
+      rustfmt_vers=0.8.3;
+      if ! rustfmt --version | grep -q $rustfmt_vers; then
```

Comment:
It is to be done with QA cargo_install script - refer to how routing 0.29.0 does it for e.g.